01-filedir.md

```diff
@@ -426,7 +426,7 @@ and we will see it in many other tools as we go on.
 > 3.  `2012-12-01/ 2013-01-08/ 2013-01-27/`
 > 4.  `original pnas_final pnas_sub`
 
-> ## FIXME {.challenge}
+> ## `ls` command construction {.challenge}
```

Comment:
On Sat, Jan 31, 2015 at 07:38:03PM -0800, Greg Wilson wrote:
> > @@ -426,7 +426,7 @@ and we will see it in many other tools as we go on.
> >  > 3.  `2012-12-01/ 2013-01-08/ 2013-01-27/`
> >  > 4.  `original pnas_final pnas_sub`
> >  
> > -> ## FIXME {.challenge}
> > +> ## `ls` command construction {.challenge}
> 
> Please go ahead and send a PR with the change (and others).

I'm happy just re-rolling my branch ;), so I don't know that we need
another PR here.  I should get to it by tonight.